.github/workflows/release.yaml

```diff
@@ -10,9 +10,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: PyPI Upload
-        uses: FeatureLabs/gh-action-pypi-upload@v2
```

Comment:
there is a small change in behavior from this script, which would check to confirm the verison number of the package matched the version number of the tagged release. see here: https://github.com/alteryx/gh-action-pypi-upload/blob/main/upload.sh#L31

i dont think it's critical we keep this check around, but just wanted to mention just in case you wanted to try to keep it in some way

Reply:
Okay, I added the version check under the "Check version match" task. It's hard to verify if this workflow will succeed without actually running it with the secrets.